contract/Makefile

```diff
@@ -0,0 +1,50 @@
+CHAINID=agoriclocal
+USER1ADDR=$(shell agd keys show user1 -a --keyring-backend="test")
+ACCT_ADDR=$(USER1ADDR)
+BLD=000000ubld
+
+ATOM_DENOM=ibc/BA313C4A19DFBF943586C0387E6B11286F9E416B4DD27574E6909CABE0E342FA
+ATOM=000000$(ATOM_DENOM)
+
+.PHONY: list
+# https://stackoverflow.com/a/73159833/7963
+list:
+	make -npq : 2> /dev/null | \
+		awk -v RS= -F: '$$1 ~ /^[^#%]+$$/ { print $$1 }'
+
+balance-q:
+	agd keys show user1 -a --keyring-backend="test"
+	agd query bank balances $(ACCT_ADDR)
+
+GAS_ADJUSTMENT=1.2
+SIGN_BROADCAST_OPTS=--keyring-backend=test --chain-id=$(CHAINID) \
+		--gas=auto --gas-adjustment=$(GAS_ADJUSTMENT) \
+		--yes -b block
+
+mint4k:
+	make FUNDS=1000$(ATOM) fund-acct
+	cd /usr/src/agoric-sdk && \
+		yarn --silent agops vaults open --wantMinted 4000 --giveCollateral 1000 >/tmp/want4k.json && \
```

Comment:
I don't know the `agops` signature off hand, but do we need to specify a denom here? The `agd tx bank ... ` command is missing it

```log
$ docker-compose exec agd make -C /workspace/contract mint4k
make: Entering directory '/workspace/contract'
make FUNDS=1000 fund-acct
make[1]: Entering directory '/workspace/contract'
agd tx bank send validator agoric16xuc6qyl09kxhk9u8hukv7dqmvtg442mmn0jhs 1000 \
  --keyring-backend=test --chain-id=agoriclocal --gas=auto --gas-adjustment=1.2 --yes -b block \
  -o json >,tx.json
Error: invalid decimal coin expression: 1000
```

Reply:
`agops vaults open` was written as a quick-and-dirty tool to test vaults, so defaulting **ATOM** was handy. This is no longer considered a feature:

 - https://github.com/Agoric/agoric-sdk/issues/8430

a3a19a7 should address the missing denom